datastrato/gravitino
# [#3864] feat(lakehouse-iceberg): add custom catalog name to  Iceberg catalog backend

### What changes were proposed in this pull request?
add custom catalog name to Iceberg catalog backend

### Why are the changes needed?

Fix: #3864 #3865 

### Does this PR introduce _any_ user-facing change?
yes, add configuration document

### How was this patch tested?
1.  create a jdbc catalog with custom iceberg-catalog-name
2.  check it takes effects in MySQL 
3.  spark connector could successfully operate under the jdbc catalog.
